pkg/tsdb/elasticsearch/client/search_request_test.go

```diff
@@ -15,7 +16,8 @@ func TestSearchRequest(t *testing.T) {
 	Convey("Test elasticsearch search request", t, func() {
 		timeField := "@timestamp"
 		Convey("Given new search request builder for es version 5", func() {
-			b := NewSearchRequestBuilder(5, interval.Interval{Value: 15 * time.Second, Text: "15s"})
+			version5, _ := semver.NewVersion("2.0.0")
```

Comment:
The test is for 5 but semver says 2.0.0?

```suggestion
			version5, _ := semver.NewVersion("5.0.0")
```

How come the test is passing? 🤔

Reply:
i guess my find-replace skills are not top-notch :)

but I think they are still failing for other reasons